locales/en-US/webextension.properties

```diff
@@ -15,14 +16,26 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+# Error Strings
 requestErrorTitle = Firefox Screenshots is out of order.
-requestErrorDetails = Your shot was not saved. We apologize for the inconvenience. Try again soon.
-connectionErrorTitle = Cannot connect to the Firefox Screenshots server.
-connectionErrorDetails = There may be a problem with the service or with your network connection.
-loginErrorDetails = Your shot was not saved. There was an error authenticating with the server.
-loginConnectionErrorDetails = There may be a problem with the service or your network connection.
-unshootablePageErrorTitle = Page cannot be screenshotted.
-unshootablePageErrorDetails = This is not a normal web page, and Firefox cannot capture screenshots from it.
+requestErrorDetails = Sorry! We couldn’t save your shot. Please try again later.
+connectionErrorTitle = We can’t connect to the Firefox Screenshots server.
+connectionErrorDetails = Please check your network connection. If you are able to connect to the Internet, there may be a temporary problem with the Firefox Screenshots service.
+loginErrorDetails = We couldn’t save your shot because we couldn’t access your account on the Firefox Screenshots server.
+loginConnectionErrorDetails = Please check your network connection. If you are able to connect to the Internet, there may be a temporary problem with the Firefox Screenshots service.
+unshootablePageErrorTitle = We can’t screenshot this page.
+unshootablePageErrorDetails = his isn’t a standard Web page, so Firefox Screenshots can’t take a screenshot of it.
 selfScreenshotErrorTitle = You can’t take a shot of a Firefox Screenshots page!
-genericErrorTitle = Firefox Screenshots went haywire.
-genericErrorDetails = Try again or take a shot on another page?
+genericErrorTitle = Whoa! Screenshots went haywire.
+genericErrorDetails = We’re not sure what just happened. Care to try again or take a shot of a different page?
+## Onboarding Strings
+tourHeaderOne = Firefox Screenshots
+tourHeaderOneSuper = Beta
+tourBodyOne = Take, save, and share screenshots without leaving Firefox.
+tourHeaderTwo = Capture Just What You Want
+tourBodyTwo = Click and drag to capture just a portion of a page.  You can also hover to highlight your your selection.
+tourHeaderThree = As You Like it
+tourBodyThree = Save your cropped shots to the Web for easier sharing, or download them to your computer. You also can click on the My Shots button to find all the shots you’ve taken.
+tourHeaderFour = Capture Windows or Entire Pages
+tourBodyFour = Select the buttons in the upper right to capture the visible area in the window or to capture an entire page. Shot Taken Push Notification
```

Comment:
Pretty annoying that github showed all this typos as fixed. Going through them now.